helm/prefect-server/values.yaml

```
@@ -0,0 +1,273 @@
+# string
+postgresql:
+  postgresqlDatabase: prefect
+  postgresqlUsername: prefect
+
+  # existingSecret configures which secret should be referenced
+  # for access to the database. If null and `useSubChart` is
+  # enabled, the secret will be generated. If using an external
+  # postgres service, this value should be set to a secret
+  # containing the key `postgresql-password`
+  existingSecret: null
+
+  # postgresqlPassword sets the password to be used if
+  # `existingSecret` is not set. This is the password for
+  # `postgresqlUsername` and will be set within the secret at
+  # the key `postgresql-password`
+  # postgresqlPassword: use-a-strong-password
+
+  # servicePort configures the port that the database should be
+  # accessed at
+  servicePort: 5432
+
+  # externalHostname defines the address to contact an externally
+  # managed postgres database instance at. This is not required if
+  # `internalPostgres` is `true`
+  externalHostname: ""
+
+  # useSubChart determines if a this chart should deploy a
+  # user-manager postgres database or use an externally managed
+  # postgres instance. If `useSubChart` is `true`, the
+  # bitnami/postgresql subchart will be deployed
+  useSubChart: true
+
+  # postgresql configuration below here is only used if using
+  # the subchart
+
+  # persistence enables a PVC that stores the database between
+  # deployments. If making changes to the database deployment, this
+  # PVC will need to be deleted for database changes to take effect.
+  # This is especially notable when the authentication password
+  # changes on redeploys.
+  # This is disabled by default because we do not recommend using
+  # the subchart deployment for production deployments.
+  persistence:
+    enabled: false
+    size: 8Gi
+
+  initdbUser: postgres
+
+  initdbScripts:
+    create_pgcrypto.sql: |
+      -- create pgcrypto extension, required for Hasura UUID
+      CREATE EXTENSION IF NOT EXISTS pgcrypto;
+      CREATE EXTENSION IF NOT EXISTS "pg_trgm";
+      SET TIME ZONE 'UTC';
+
+# prefectConfig allows override of prefect_server/config.toml values
+# Values placed here will result in environment variable definitions
+# starting with "PREFECT_SERVER__" and will be passed to TODO
+prefectConfig:
+  env: "local"
+  debug: "false"
+  queued_runs_returned_limit: "25"
+  hasura:
+    execute_retry_seconds: "10"
+  logging:
+    level: "DEBUG"
+    format: "[%(asctime)s] %(levelname)s - %(name)s | %(message)s"
+  services:
+    scheduler:
+      scheduler_loop_seconds: "300"
+    lazarus:
+      resurrection_attempt_limit: "3"
+  telemetry:
+    enabled: "true"
+
+
+# hasura configures the Prefect hasura deployment and service
+# which creates a graphql api from the postgres database
+hasura:
+  port: 3000
+
+  # hasura.image configures the docker image used for hasura
+  # and is the only image in the chart that is not hosted by
+  # Prefect
+  image:
+    repository: hasura/graphql-engine
+    version: v1.3.0
```

```suggestion
    name: hasura/graphql-engine
    tag: v1.3.0
```

`name` and `tag` are more common here (and `tag` is more accurate than `version`.